libut/Makefile

```diff
@@ -1,33 +1,41 @@
-OBJS=libut.a
-all: $(OBJS) 
+SHELL=/usr/sh
+VERSION=2.0.1
+MAJOR=2
+SOURCES=$(shell echo src/*.c)
+HEADERS=$(shell echo include/*.h)
```

Comment:
Please just list these files manually, instead of relying on shell wildcard expansion to do the right thing. (Which it *wouldn't*, if someone's checkout had untracked files in it, right?)